eng/CredScanSuppression.json

```diff
@@ -44,7 +44,8 @@
         "liilef#$DdRGSa_ewkjh",
         "kt#_gahr!@aGERDXA",
         "goose",
-        "_password"
+        "_password",
+        "YmFzZTY0X3BsYWNlaG9sZGVy"
```

Comment:
To avoid adding a new one here, can you encode that on the fly?

Reply:
Similar to what I did here https://github.com/Azure/azure-sdk-for-js/pull/16700

Reply:
This will be saved in recording files so we want a fixed one right?

Reply:
I see, that makes sense.

Reply:
Yes ideally this will be be a fixed value and it should match an existing value if possible.

Reply:
I will do a follow-up PR to unify fake passwords